ledger-core/v2/virtual/integration/constructor_api_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package small
+
+import (
+	"testing"
+
+	"github.com/ThreeDotsLabs/watermill/message"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/insolar/assured-ledger/ledger-core/v2/instrumentation/inslogger"
+	"github.com/insolar/assured-ledger/ledger-core/v2/virtual/integration/utils"
+)
+
+func Test_API_Create(t *testing.T) {
+	server := utils.NewServer(t)
+	ctx := inslogger.TestContext(t)
+
+	server.PublisherMock.Checker = func(topic string, messages ...*message.Message) error {
+		// verify and decode incoming message
+		assert.Len(t, messages, 1)
+
+		server.SendMessage(ctx, messages[0])
+
+		return nil
+	}
+
+	code, byteBuffer := server.CallAPICreateWallet()
+	if !assert.Equal(t, 200, code) {
```

Comment:
When it can be not equal to 200?

Reply:
When we can't parse input

Reply:
In Creation API we can't have it (if everything is ok)

Reply:
If we can't have it, maybe test has to fail?

Reply:
assert will fail and we will log response

Reply:
ok